org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4viz/VizGUI.java

```diff
@@ -1411,7 +1670,109 @@ private Runner doNext() {
             OurDialog.alert(frame, "Cannot display the next solution since the analysis engine is not loaded with the visualizer.");
         } else {
             try {
-                enumerator.compute(xmlFileName);
+                enumerator.compute(new String[] { // [HASLab]
```

Comment:
the computer is now expecting an array so that different iteration operations can be called